os/services/rpl-border-router/native/slip-dev.c

```diff
@@ -316,7 +328,30 @@ slip_empty()
   return slip_packet_end == 0;
 }
 /*---------------------------------------------------------------------------*/
-void
+static ssize_t
+write_slowly(int fd, const void *buf, size_t count, int inter_octet_delay)
+{
+  if(inter_octet_delay == 0) {
+    return write(fd, buf, count);
+  } else {
+    int n = 0;
+    int write_status;
+    while(n < count) {
+      write_status = write(fd, buf + n, 1);
```

Comment:
It would be nice to say something like "writing data one byte at a time" here.

Reply:
Nice idea. Done.